docs/questions/sharing/visualizations/progress-bar.md

```diff
@@ -4,6 +4,100 @@ title: Progress bars
 
 # Progress bars
 
-**Progress bars** are for comparing a single number to a goal value that you set. Open up the settings for your progress bar to choose a value for your goal, and Metabase will show you how far away your question's current result is from the goal.
+**Progress bars** are for comparing a single number to a goal value that you set.
 
-![Progress bar](../../images/progress.png)
+## Example use cases
+
+- KPI tracking
+
+  ![Progress bar KPI](../../images/progress-kpi.png)
+
+- Percent completion
+
+  ![Progress bar % completion](../../images/progress-percent.png)
+
+## Progress bar prerequisites
+
+To create a progress bar you'll need:
+
+- A query that returns a single number, like "Sum of order quantity". Progress bar doesn't work with breakouts.
+
+- A goal value. The goal value should be non-negative. Currently, Metabase only supports setting a static goal (you can't set a goal based on another query).
```

Comment:
Metabase will not complain if you set a negative goal but it will not show progress towards it from more-negative numbers, which makes the bar useless